dist/tools/zep_dispatch/l2util.c

```diff
@@ -0,0 +1 @@
+../../../sys/net/link_layer/l2util/l2util.c
```

Comment:
I would prefer to have these in the Makefile, similar to how `ieee802154.c` is included in the build now. Is there anything against that?

Reply:
fine with me